hardware_interface/src/mock_components/generic_system.cpp

```diff
@@ -26,22 +26,12 @@
 #include <vector>
 
 #include "hardware_interface/component_parser.hpp"
+#include "hardware_interface/helpers.hpp"
```

Comment:
Can we add something like this?
```suggestion
#if CPP_VER<=17
#include "hardware_interface/helpers.hpp"
#endif
```

Reply:
...in a follow-up ticket

Reply:
The goal is to add the old code of parsing with `std::from_chars` protected by CPP version protection. If a higher version then C++17 (this flag `#if __cplusplus > 201402L`) is used then we should you this standard; otherwise we use the here implemented version.